src/components/scores/export.cjsx

```diff
@@ -101,12 +96,12 @@ ScoresExport = React.createClass
       tryToDownload: false
       downloadUrl: downloadUrl
       lastExported: lastExported
-      forceDownloadUrl: null
 
     @setState(downloadState)
+    @downloadCurrentExport(downloadUrl)
 
-  downloadCurrentExport: ->
-    @setState(tryToDownload: true, downloadedSinceLoad: true)
+  downloadCurrentExport: (url) ->
+    document.getElementById('downloadExport').src = url;
```

Comment:
if we are doing this, i would recommend setting the `src` declaratively in the `render` function to be more `React`ish.  meaning take out line 101 and this function and in `render` line 150 where `downloadExport` is, just say

```
<iframe id="downloadExport" src={downloadUrl}></iframe>
```